### DISCUSSION TITLE
React-native support

### DISCUSSION THREAD

**Discussion by durgaprasadksv** [tinacms/tinacms#4505] [Help]
Hello folks, we are evaluating tinacms for our product and curious if react-native is supported. We plan to use MDX as our primary content production mode. 

Thanks

**Comment by scottgallant** [comment]
Hi @durgaprasadksv I believe this is possible but I'd like to learn more about your project. Are you free for a quick call this week? If so, you can book me [here](https://meetings-eu1.hubspot.com/scott15).
